subgraphs/makerdao/src/mapping.ts

```diff
@@ -445,20 +447,28 @@ export function handleDogBark(event: BarkEvent): void {
   const due = event.params.due; //including interest, but not penalty
 
   const market = getMarketFromIlk(ilk)!;
-  const storeID = clip.toHexString().concat("-").concat(id.toString());
-
-  // remove borrowed amount from borrowed balance
-  // collateral/tvl update is taken care of when it exits vat
-  // via the slip() function/event
+  const token = getOrCreateToken(market.inputToken);
```

Comment:
Some duplicate logic in `handleDogBark()` and `handleCatBite()` but it is not too much and there would be a lot of variables if you wanted to make a helper function so I will leave it.

Also, the naming schema in makerdao is so strange

Reply:
Yep, I am on the same boat. There are just too many variables and difference. The temporary data store entities are also different, FlipBidStore for cat and ClipBidStore for dog. Because it is two stage auction for cat (the old collateral auction contract, one stage for dog). I lean toward leaving it as it is.

The strange naming scheme is a hallmark of makerdao code. 😁

Reply:
maker code seems like a rabbit hole that will take me a while to get through. You do a good job 🙏